Flutter/calendar/drag-drop.md

```diff
@@ -125,7 +125,7 @@ void dragUpdate(AppointmentDragUpdateDetails appointmentDragUpdateDetails) {
 [targetResource](https://pub.dev/documentation/syncfusion_flutter_calendar/latest/calendar/AppointmentDragEndDetails/targetResource.html) - Get the target resource details.
 
 {% tabs %}
-{% highlight Dart %}
+{% highlight dart hl_lines="9 10" %}
```

Comment:
Highlight method also on below topics

https://help.syncfusion.com/flutter/calendar/appointment-resizing#onappointmentresizestart
https://help.syncfusion.com/flutter/calendar/appointment-resizing#onappointmentresizeupdate
https://help.syncfusion.com/flutter/calendar/appointment-resizing#onappointmentresizeend

Reply:
highlighted the method also.

Reply:
![image](https://user-images.githubusercontent.com/54974849/171105771-9f58d52c-2b5f-4a4e-b582-250a2a663065.png)
![image](https://user-images.githubusercontent.com/54974849/171105914-d5fa0892-03eb-4a5d-8e56-4af1ff252e72.png)
![image](https://user-images.githubusercontent.com/54974849/171105963-57a5ab3c-ee57-4f06-8848-8a2ba4e96705.png)
~ already updated in pull request.